nbval/kernel.py

```diff
@@ -75,14 +75,15 @@ class RunningKernel(object):
     this class.
 
     """
-    def __init__(self, kernel_name, cwd=None):
+    def __init__(self, kernel_name, startup_timeout, cwd=None):
```

Comment:
I'm not sure if anyone is using nbval as a library, but just in case someone is: 
```suggestion
    def __init__(self, kernel_name, cwd=None, startup_timeout=60):
```

Reply:
Ah, didn't think of that, thanks for the suggestion!